pkgs/development/tools/vulkan-validation-layers/default.nix

```diff
@@ -20,8 +67,8 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [
-    glslang
-    spirv-headers
+    localGlslang
+    localGlslang.localSpirv-headers
```

Comment:
If you change back the API, as I think you should, don't forget this place. Note it shouldn't trigger rebuilds, but please check it all.